### PR TITLE
fix up flaky old tests, and move from Travis-CI to actions

### PR DESCRIPTION
**Pull request by ryanpetrello**
this removes some old flaky tests, and also runs py36 through py39 tests via Github Actions

### CODE REVIEW

File: pecan/tests/scaffold_builder.py [hunk 1]
```diff
@@ -117,27 +117,6 @@ def poll_http(self, name, proc, port):
             finally:
                 proc.terminate()
 
-    class TestGunicornServeCommand(TestThirdPartyServe):
-
-        def test_serve_from_config(self):
```
Comment (ryanpetrello) [inline pecan/tests/scaffold_builder.py:122]: This test is busted, and I don't know why.

File: setup.py [hunk 2]
```diff
@@ -67,8 +67,10 @@
         'Operating System :: POSIX',
         'Programming Language :: Python',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.5',
```
Comment (ryanpetrello) [inline setup.py:70]: 3.5 support was removed awhile ago

**Comment by ryanpetrello** [comment]
Travis is probably going to complain until this is merged, but here's a matrix of these tests passing in my fork:

https://github.com/ryanpetrello/pecan/actions/runs/778162781

File: .github/workflows/run-tests.yml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    - cron:  '0 12 * * *'  # daily ~8am est
+  workflow_dispatch:
+  pull_request:
+
+jobs:
+  unit-tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python: [3.6, 3.7, 3.8, 3.9]
```
Comment (alfredodeza) [inline .github/workflows/run-tests.yml:14]: looks like you can add another dimension to this matrix with the tox command to run, e.g.:

```yaml
    command: ["py", "scaffolds"]
```

That would avoid having the (mostly) identical job repetition

**Review by alfredodeza — APPROVED** [review]

**Pull request merged**